src/components/SearchModal/CurrencyList/index.tsx

```diff
@@ -166,9 +163,9 @@ export function CurrencyRow({
               <TokenSafetyIcon warning={warning} />
             </WarningContainer>
           </Row>
-          <ThemedText.DeprecatedDarkGray ml="0px" fontSize="12px" fontWeight={300}>
+          <ThemedText.BodySecondary ml="0px" fontSize="12px" fontWeight={485}>
```

Comment:
`BodySecondary` already has `fontWeight: 485`

However `LabelMicro` has both `fontWeight: 485` and `fontSize:12px`